### DISCUSSION TITLE
Arctos Collection Dashboard Ideas

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4031] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to plan for a collection dashboard in Arctos. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you would like to see in a collection dashboard.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Nicole Volden - New Mexico Museum of Natural History and Science Paleo/Geo

My first question is - what do you mean by a collection dashboard?

**Reply by Jegelewicz** [reply]
See https://github.com/ArctosDB/arctos/issues/3983 for examples
